index.html

```diff
@@ -0,0 +1,173 @@
+      </div>
+    </div>
+  </div>
+</section>
+
+
+  <!-- Main -->
+  <main>
+    <!-- 섹션 1: 인기 상품 -->
+    <section class="section section--hot">
+      <div class="rail">
+        <div class="grid-2">
+          <div class="card card--image">
+            <!-- 좌측 일러스트 (T셔츠/하트 등) -->
+            <img
+              src="./assets/images/Img_home_01.svg"
+              alt="인기 상품 미리보기"
+              loading="lazy"
+            />
+          </div>
+          <div class="card card--text">
+            <em class="eyebrow">Hot item</em>
+            <h2 class="h2">인기 상품을<br />확인해 보세요</h2>
+            <p class="desc">가장 HOT한 중고거래 물품을<br> 판다마켓에서 확인해 보세요.</p>
+          </div>
+        </div>
+      </div>
+    </section>
+
+    <!-- 섹션 2: 검색 -->
+    <section class="section section--search">
+      <div class="rail">
+        <div class="grid-2">
+          <div class="card card--text">
+            <em class="eyebrow">Search</em>
+            <h2 class="h2">구매를 원하는<br />상품을 검색하세요</h2>
+            <p class="desc">구매하고 싶은 물품을 검색해서<br>쉽게 찾아보세요.</p>
+          </div>
+          <div class="card card--image">
+            <img
+              src="./assets/images/Img_home_02.svg"
+              alt="검색 일러스트"
+              loading="lazy"
+            />
+          </div>
+        </div>
+      </div>
+    </section>
+
+    <!-- 섹션 3: 등록 -->
+    <section class="section section--register">
+      <div class="rail">
+        <div class="grid-2">
+          <div class="card card--image">
+            <img
+              src="./assets/images/Img_home_03.svg"
+              alt="상품 등록 일러스트"
+              loading="lazy"
+            />
+          </div>
+          <div class="card card--text">
+            <em class="eyebrow">Register</em>
+            <h2 class="h2">판매를 원하는<br />상품을 등록하세요</h2>
+            <p class="desc">어떤 물건이든 판매하고 싶은 상품을<br>쉽게 등록해 보세요.</p>
+          </div>
+        </div>
+      </div>
+    </section>
+
+    <!-- 신뢰 배너 (하단 하늘색 섹션) -->
+    <section class="section section--trust">
+      <div class="rail">
+        <div class="trust">
+          <h2 class="h2">믿을 수 있는<br />판다마켓 중고 거래</h2>
+          <div class="trust__image">
+            <img
+              src="./assets/images/Img_home_bottom.svg"
+              alt=""
+              loading="lazy"
+            />
+          </div>
+        </div>
+      </div>
+    </section>
+
+    <!-- Footer -->
+    <footer class="footer">
+    <div class="rail">
+      <!-- 왼쪽: 저작권 정보 -->
+      <div class="footer__left">©codeit · 2024</div>
+
+      <!-- 가운데: 링크들 -->
+      <div class="footer__center">
+        <a href="./privacy.html">Privacy Policy</a>
+        <a href="./faq.html">FAQ</a>
+      </div>
+
+      <!-- 오른쪽: SNS 아이콘들 -->
+      <div class="footer__sns">
+        <a href="https://facebook.com" target="_blank" rel="noopener">
+          <img src="./assets/icons/ic_facebook.svg" alt="Facebook" width="20" height="20" />
+        </a>
```

Comment:
💊 제안  
이 영역은 로고들의 모음이라고 할 수 있습니다. 관련된 요소들끼리 묶인 목록이니 의미록적인 마크업을 작성한다면 list 관련 태그로 작성하시는 것을 추천드립니다.

```suggestion
      <ul class="footer__sns">
        <li>
          <a href="https://facebook.com" target="_blank" rel="noopener">
            <img src="./assets/icons/ic_facebook.svg" alt="Facebook" width="20" height="20" />
          </a>
        </li>
```